### PR TITLE
Move tests to correct sub-package

### PR DESCRIPTION
**Pull request by Hinton**
## Type of change
- [ ] Bug fix
- [ ] New feature development
- [x] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other

## Objective

We previously split up the jslib repo into multiple modules but the tests remained in the root project. With npm v7 having support for workspaces, I sent some time cleaning it up and moving the tests to the correct packages.

This should also allow us to properly build each package independently and use the built files vs depending on the typescript source. Which I believe will help alleviate many of the issues we've encountered.

## Before you submit
- [x] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] I have added **unit tests** where it makes sense to do so (encouraged but not required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

File: common/karma.conf.js [hunk 1]
```diff
@@ -68,31 +78,6 @@ module.exports = (config) => {
                 reasons: true,
                 errorDetails: true,
             },
-            devtool: 'inline-source-map',
-        },
-
-        detectBrowsers: {
```
Comment (Hinton) [inline common/karma.conf.js:74]: TODO: Should bring this back

File: common/spec/importers/firefoxCsvImporter.spec.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
-import { FirefoxCsvImporter as Importer } from 'jslib-common/importers/firefoxCsvImporter';
+import { FirefoxCsvImporter as Importer } from '../../src/importers/firefoxCsvImporter';
```
Comment (djsmith85) [inline common/spec/importers/firefoxCsvImporter.spec.ts:1]: Do we have to do this? I'd prefer it staying the way it is. The clients using jslib also work with referrering to jslib-common package

Comment (Hinton) [reply common/spec/importers/firefoxCsvImporter.spec.ts]: Added `src` as an alias.

**Comment by MGibson1** [comment]
> 

Do you happen to have an issue/commit to link to?

**Comment by MGibson1** [comment]
This was done in #744

**Pull request closed without merging**